TGEngine/drawlib/Beziercurve.cpp

```diff
@@ -1,5 +1,37 @@
 #include "Beziercurve.hpp"
 
-void drawBeziercurve(TGVertex ver1, TGVertex ver2, TGVertex ver3, int max_verticies, VertexBuffer* verticeis) {
-	
+static TGVertex midpoint(TGVertex a, TGVertex b, float t)
```

Comment:
No static functions please, even though they are used only once ... this could change.

```suggestion
TGVertex midpoint(TGVertex a, TGVertex b, float t)
```
And please move this function to `TGEngine/util/Math.cpp`